proposals/lip-0063.md

```diff
@@ -268,6 +268,13 @@ Lisk Core v4 will use the following modules, where the given order defines the m
 8. [Dynamic Block Reward](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0071.md)
 9. [Legacy](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0050.md)
 
+##### Registration of Interoperable Modules
+
+The interoperability modules requires to define the registration order of all interoperable modules. This is needed to define the order for iterating through all interoperable modules, e.g., [here](https://github.com/LiskHQ/lips/blob/87dc7bc4294e20fe74b5e713e999c7448b5ad26b/proposals/lip-0049.md?plain=1#L259). The order is as follows:
```

Comment:
Also note that link given by you is permanent, so in future it might point to outdated code. You could be more general by saying something like: "this is needed to define the order of modules to execute interoperability hooks in [apply](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0049.md#apply)".

Reply:
I used a permanent link on purpose. This way, it is ensured that the reader is linked to the specific line I meant instead of section where they need to search for it. Also the link to the apply section could be removed in a future version of the document.
But if you strongly prefer to no use a permanent link, I can change it.

Reply:
LIPs should be immutable once we release Lisk Core 4.0 and Lisk SDK 6.0. Changes for future releases would require new LIPs. I think it is therefore fine to link the general `apply` logic as Sergey suggest. I think this is also nicer compared to using a permanent link.